tephra-core/src/main/java/co/cask/tephra/TxConstants.java

```diff
@@ -310,4 +310,16 @@
       { DefaultSnapshotCodec.class, SnapshotCodecV2.class, SnapshotCodecV3.class, SnapshotCodecV4.class };
   }
 
+  /**
+   * Configuration for transaction log edit entries
+   */
+  public static final class TransactionLog {
+    /**
+     * Key used to denote the number of entries appended.
+     */
+    public static final String NUM_ENTRIES_APPENDED = "entries.append.count";
+    public static final String VERSION_KEY = "version";
+    public static final String CURRENT_VERSION = "v2";
```

Comment:
Since we already have the key named version, I think we can just keep versions as integers. This will help us do version comparisons later if needed.